src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -1278,7 +1288,7 @@
 examples:
   - name: List application gateway WAF policy custom rules. (autogenerated)
     text: |
-        az network application-gateway waf-policy custom-rule list --policy-name MyPolicy --resource-group MyResourceGroup
+        az network application-gateway waf-policy custom-rule list --policy-name MyPolicy --resource-group MyResourceGroup --subscription MySubscription
```

Comment:
Have one concern to add example with --subscription. 
If there is an  example without subscription, this one is a patch. I am fine.
But if with only one example, I will prefer previous one.
because the new one will bring some trouble to cx with --subscription

Reply:
```suggestion
        az network application-gateway waf-policy custom-rule list --policy-name MyPolicy --resource-group MyResourceGroup
```